pkg/resources/network_policy.go

```diff
@@ -2,8 +2,10 @@ package resources
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
-	"log"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```

Comment:
when we run the linter it should be marked as nongofumpted (please run make pre-push).

Reply:
Sorry, forgot about this. done.